fastchat/serve/api_provider.py

```diff
@@ -131,6 +131,7 @@ def openai_api_stream_iter(
         temperature=temperature,
         max_tokens=max_new_tokens,
         stream=True,
+        extra_headers={"Accept": "text/event-stream"},
```

Comment:
could you explain why we need this?

Reply:
When I used project [oneapi](https://github.com/songquanpeng/one-api) (OpenAI interface conversion & management & distribution system), some interfaces needs to a header `"Accept": "text/event-stream"` to accept stream iteration. And Fastchat doesn't support none-streaming so if this line is not added, it couldn't accept stream answers from third-party interfaces. This header still works fine for standard openai-api.